### DISCUSSION TITLE
How to make the window position set to the bottom right corner of the screen

### DISCUSSION THREAD

**Discussion by wkl007** [tauri-apps/tauri#4698] [Q&A] [answered]
At the same time, the screen zoom position is always in the lower right corner. Please help me, thank you.

Example: Jetbrains Toolbox.

![image](https://user-images.githubusercontent.com/23228335/179388605-223f0018-ba10-4be3-910a-6547763992f2.png)

**Comment by wkl007** [comment, marked as answer]
This plugin solved my problem.

https://github.com/JonasKruckenberg/tauri-plugin-positioner

**Reply by JakubGluszek** [reply]
Thanks for this.
